numpy/distutils/tests/test_ccompiler_opt_conf.py

```diff
@@ -161,9 +161,9 @@ def test_extra_checks(self, error_msg, search_in, feature_name, feature_dict):
 class TestConfFeatures(unittest.TestCase):
     def __init__(self, methodName="runTest"):
         unittest.TestCase.__init__(self, methodName)
-        self.setup()
+        self._setup()
 
-    def setup(self):
+    def _setup(self):
```

Comment:
Since the init calls it explicitly, this seemed right.  There is only one test, so it doesn't matter anyway, but I suspect the intention isn't to rerun it on each test method.

Reply:
And hopefully we can remove distutils in a few years :)